src/data-access/repositories/mongo/mongo-database.ts

```diff
@@ -4,8 +4,10 @@ import { DatabaseNotConnectedException } from '../../../model/exceptions/databas
 import { MongoId } from './mongo-entity-converter'
 
 // TODO: Move to ENV variables
-const MONGO_CONNECTION_STRING: string = 'mongodb://localhost:3001'
-const MONGO_DB_NAME: string = 'meteor'
+// const MONGO_CONNECTION_STRING: string = 'mongodb://localhost:3001'
+// const MONGO_DB_NAME: string = 'meteor'
+const MONGO_CONNECTION_STRING: string = 'mongodb://localhost:27017?directConnection=true'
+const MONGO_DB_NAME: string = 'sofie'
```

Comment:
I gather you mistakenly commited changes for mongo-database.

Reply:
Yup, that was a mistake. And it's not even the first time...

Reply:
It's been removed